tasks.py

```diff
@@ -3,51 +3,59 @@
 Task execution tool & library
 """
 
+import copy
 import os
 import platform
 import re
 import shutil
 import subprocess
 import sys
 from datetime import datetime
-from logging import basicConfig, getLogger
+from logging import DEBUG, basicConfig, getLogger
 from pathlib import Path
 from typing import Union
 
 import docker
+import requests
 from bumpversion.cli import main as bumpversion
 from invoke import task
 
-from easy_infra import __project_name__, __version__, constants, utils
+from easy_infra import __project_name__, __version__, config, constants, utils
 from tests import test as run_test
 
-if platform.machine() == "arm64":
-    PLATFORM: Union[str, None] = "linux/arm64/v8"
+# TODO: Does this work on intel?
```

Comment:
I need someone with an intel-based system to run this @digeseiso / @seanctech

Reply:
@JonZeolla I built the image successfully on my Intel-based mac